### ISSUE TITLE
OACMPDM1 small texture issue next to the teleporter

### ISSUE THREAD

**Issue by EddieBrrrock** [NeonKnightOA/oacmpvol1#29]
![Capture d’écran 2022-04-12 à 10 58 50](https://user-images.githubusercontent.com/99912467/162923017-b91222cc-729e-496e-910d-e06fbd00dfcc.png)

**Comment by EddieBrrrock** [comment]
Sorry for the zoom, it's the wall at the right of the teleporter when you looking in the same way with the liz statue.

**Comment by EddieBrrrock** [comment]
After a confirmation I think I can close this issue because I think the problem is the entire wall and it refer to this issue : https://github.com/NeonKnightOA/oacmpvol1/issues/24

**Comment by EddieBrrrock** [comment]
z_oacmp1v4_beta-220411

**Comment by NeonKnightOA** [comment]
This is because of a shader file that wasn't incorporated when the map was being compiled.

It uses common/skip, which isn't present in the base OA. So it had to be added to a separate file. And now I've got to check all files in case I've used common/skip in any of them. :facepalm:

**Comment by NeonKnightOA** [comment]
Removing the hint/skip brushes possibly did the trick. Test please!

**Comment by The-Gig** [comment]
Wouldn't removing hint brushes worsen map performances?

**Comment by NeonKnightOA** [comment]
Unfortunately, for this map it was the solution.

**Comment by The-Gig** [comment]
_Waiting for an update to the map to test it,_ as it turned out there was no need to place common/skip and common/sky_space in newcommon.shader considering they were already in pul1duel-missing.shader and oasky.shader, respectively.

PS: Maybe the hint brushes in this map were a bit off after you made a certain edit, as older versions didn't have the glitch?
https://github.com/NeonKnightOA/oacmpvol1/issues/24#issuecomment-1102530791

**Comment by NeonKnightOA** [comment]
I thought I've uploaded the map yesternight.

**Comment by The-Gig** [comment]
Yes, but if I understood correctly, yesterday you built that with common/skip in newcommon.shader, where it should not be. Just to be sure, I'd prefer to see if it works with the common/skip from 0.8.8.
Are any of the other shaders in newcommon.shader actually used in this map?

**Comment by NeonKnightOA** [comment]
Actually, no.

**But I'm more interested in seeing if the problem is solved or not.**

**Comment by The-Gig** [comment]
I downloaded your .bsp and .aas file, and tried the map. I didn't notice any of the glitches!

Furthermore, I think I managed to restore the hint brushes!
I opened in Radiant the last version of the .map which didn't have the glitches, https://github.com/NeonKnightOA/oacmpvol1/commit/e1f552be3f4c277b93c7175520c8c978906d5c5e (the last one before some coordinates where somehow messed up, as mentioned here https://github.com/NeonKnightOA/oacmpvol1/issues/24#issuecomment-1102530791), then I selected all the hint brushes I managed to find (there are a lot of them!) and copy-pasted them into your latest version of the .map (https://github.com/NeonKnightOA/oacmpvol1/commit/f586539feded266babfbfbeaeb4e609baf203f05). I compiled it and it looks to me it still doesn't produce the glitch and it makes a bit of a better job with vis optimization than without hint brushes.
Here it is, give it a try compiling it with your own settings, and let's see the results...
[oacmpdm1-hinttest.zip](https://github.com/NeonKnightOA/oacmpvol1/files/10980022/oacmpdm1-hinttest.zip)

**Comment by NeonKnightOA** [comment]
https://mega.nz/file/bMZkRJBa#J_rUm2nz2SpiMFl6kBY0eMceeCLOJ1PLfnz_-CQRzCw
New test version.
